feature/bgp/gracefulrestart/otg_tests/exrr/exrr_test.go

```diff
@@ -0,0 +1,1465 @@
+
+					gnoiClient := dut.RawAPIs().GNOI(t)
+					bgpReq := &bpb.ClearBGPNeighborRequest{
+						Mode: bpb.ClearBGPNeighborRequest_HARD_RESET,
+					}
+
+					if _, err := gnoiClient.BGP().ClearBGPNeighbor(context.Background(), bgpReq); err != nil {
+						t.Fatalf("Failed to clear BGP neighbor: %v", err)
+					}
+
+					validateExrr(t, flowsWithNoERR, flowsWithNoLoss, bgpGracefulRestartConfigParams)
+
+					checkBgpStatus(t, dut, routeCount)
+				}
+			},
+		},
+		{
+			name: "7_Administrative_Reset_Notification_Received_By_DUT_Hard_Reset",
+			fn: func(t *testing.T) {
+				cfgplugins.ApplyExtendedRouteRetention(t, dut, bgpGracefulRestartConfigParams)
+				ate.OTG().StartTraffic(t)
+
+				t.Log("Send Graceful Restart Trigger from OTG to DUT")
+				ate.OTG().SetControlAction(t, createGracefulRestartAction(t, peers, triggerGrTimer, "hard"))
+
+				validateExrr(t, flowsWithNoERR, flowsWithNoLoss, bgpGracefulRestartConfigParams)
+
+				checkBgpStatus(t, dut, routeCount)
+			},
+		},
+		{
+			name: "8_Additive_Policy_Application",
+			fn: func(t *testing.T) {
+				d := &oc.Root{}
+				rp := d.GetOrCreateRoutingPolicy()
+
+				setLP := rp.GetOrCreatePolicyDefinition("SET-LP")
+				setLPstmt, err := setLP.AppendNewStatement("10")
+				if err != nil {
+					t.Errorf("Error while creating new statement %v", err)
+				}
+				setLPstmt.GetOrCreateActions().PolicyResult = oc.RoutingPolicy_PolicyResultType_ACCEPT_ROUTE
+				setLPstmtCommunitySet := setLPstmt.GetOrCreateConditions().GetOrCreateBgpConditions().GetOrCreateMatchCommunitySet()
+				setLPstmtCommunitySet.SetCommunitySet("TEST-IBGP")
+				setLPstmt.GetOrCreateActions().GetOrCreateBgpActions().SetSetLocalPref(200)
+				setLPstmt.GetOrCreateActions().GetOrCreateBgpActions().SetSetMed(oc.UnionUint32(med150))
+				if deviations.BGPSetMedActionUnsupported(dut) {
+					setLPstmt.GetOrCreateActions().GetOrCreateBgpActions().SetSetMedAction(oc.BgpPolicy_BgpSetMedAction_SET)
+				}
+
+				gnmi.Update(t, dut, gnmi.OC().RoutingPolicy().Config(), rp)
+
+				cfgplugins.ApplyExtendedRouteRetention(t, dut, bgpGracefulRestartConfigParams)
+
+				ate.OTG().StartTraffic(t)
+
+				t.Log("Send Graceful Restart Trigger from OTG to DUT")
+				ate.OTG().SetControlAction(t, createGracefulRestartAction(t, peers, triggerGrTimer, "none"))
+
+				validateExrr(t, flowsWithNoERR, flowsWithNoLoss, bgpGracefulRestartConfigParams)
+
+				checkBgpStatus(t, dut, routeCount)
+			},
+		},
+		{
+			name: "9_Default_Reject_Behavior",
+			fn: func(t *testing.T) {
+				cfgplugins.DeleteExtendedRouteRetention(t, dut, bgpGracefulRestartConfigParams)
+
+				t.Log("Stop BGP on the ATE Peer")
+				stopBgp := gosnappi.NewControlState()
+				stopBgp.Protocol().Bgp().Peers().SetPeerNames(peers).
+					SetState(gosnappi.StateProtocolBgpPeersState.DOWN)
+				ate.OTG().SetControlState(t, stopBgp)
+
+				checkBgpStatusDown(t, dut)
+
+				time.Sleep(triggerGrTimer)
+
+				sendTraffic(t, ate)
+				confirmPacketLoss(t, ate, flowsWithNoERR)
```

Comment:
Why only "flowsWithNoERR"? We are expecting similar behavior even for flowsWithNoLoss. Both flows types must fail